src/ignitus-CoreTeam/Components/Team.js

```diff
@@ -8,23 +8,47 @@ import loader from "../../ignitus-Assets/Images/loader.gif";
 class CoreTeam extends React.Component {
   constructor(props) {
     super(props);
+    this.getSocialMediaIcons = this.getSocialMediaIcons.bind(this);
+    this.hideSocialMediaIcons = this.hideSocialMediaIcons.bind(this);
   }
 
-  render() {
+  getSocialMediaIcons = (index) => {
+    let cardItem = document.getElementById(`member-${index}`);
+    cardItem.classList.remove("display-none")
+    cardItem.classList.add("display-flex");
+  }
+
+  hideSocialMediaIcons = (index) => {
+    let cardItem = document.getElementById(`member-${index}`);
+    cardItem.classList.remove("display-flex")
+    cardItem.classList.add("display-none");
+  }
 
-    const teamHTML = AVATARS.map(item => (
+  render() {
+    const teamHTML = AVATARS.map((item, index) => (
       <div
         key={item.title}
         className="col-6 col-sm-4 col-md-3 avatar-wrapper text-center p-3"
       >
-        <div className="avatar bg-white p-3" id="team-avatar-card">
+        <div 
+          className="avatar bg-white p-3" 
+          id="team-avatar-card" 
+          onMouseOver={(e) => this.getSocialMediaIcons(index)}
+          onMouseOut={e => this.hideSocialMediaIcons(index)}
+        >
           <img
             className="rounded-circle mw-100 w-75"
             src={item.img}
             alt={`avatar ${item.name}`}
           />
           <div className="title-name mt-2">{item.title}</div>
           <div className="title-description">{item.description}</div>
+          <div id={`member-${index}`} class="display-none">
+            {item.facebook && <a href={item.facebook} className="team-sm-icon-color"><i class="fa fa-facebook-square"></i></a>}
```

Comment:
Social links we need to mention is only angellist and LinkedIn for all members. Can you please remove facebook and replace it with angellist along with Linkedin.